Tests/SonarScanner.MSBuild.Common.Test/MsBuildPathSettingsTests.cs

```diff
@@ -270,14 +290,36 @@ public void GetGlobalTargetsPaths_WhenProgramFilesNotEmpty_ReturnsExpectedPaths(
                 "C:\\Program\\MSBuild\\15.0\\Microsoft.Common.Targets\\ImportBefore");
         }
 
-        private static Func<bool> IsWindows(bool result) => () => result;
-        private static Func<string, bool> DirExists(bool result) => path => result;
-        private static Func<Environment.SpecialFolder, Environment.SpecialFolderOption, string> GetFolderPath(params (Environment.SpecialFolder, string)[] paths) =>
-            (folder, option) =>
-            {
-                // Bug #681 - the Create option doesn't work on some NET Core versions on Linux
-                option.Should().NotBe(Environment.SpecialFolderOption.Create);
-                return paths.First(p => p.Item1 == folder).Item2;
-            };
+        private MsBuildPathSettings MsBuildPathSettings(
+            (Environment.SpecialFolder, string)[] paths,
```

Comment:
Could be an `paramsarray`

Reply:
I tried to change, and then I reverted: `params` requires `path` to be placed at the end of the method signature. That order of parameters would be confusing for me and would also force me to change the order of parameters for the other overload of the method, which takes a single `string` path for consistency.